utils/record/record.go

```diff
@@ -274,34 +272,46 @@ func (e *EventRecorderAdapter) sendNotifications(notificationsAPI api.API, objec
 
 	objMap, err := toObjectMap(object)
 	if err != nil {
-		return err
+		return []error{err}
 	}
 
 	emptyCondition := hash("")
 
+	// We should not return in these loops because we want other configured notifications to still send if they can.
+	errors := []error{}
```

Comment:
Not sure if I am understanding this correctly but I can't find where this list of `errors` is returned..